Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
we should probably switch to Rust edition 2021, which has the resolver 2.0 enabled by default I think

Reply:
Every `Cargo.toml` is Rust 2021 but by default, virtual workspace is using `resolver = "1"` for some reason. I had a warning before each and `cargo` recommended this